content/master/cli/command-reference.md

```diff
@@ -341,9 +516,8 @@ converts a Crossplane resource to a new version or kind.
 Use the `crossplane beta convert` command to convert an existing
 [ControllerConfig]({{<ref "../concepts/providers#controller-configuration">}})
 to a [DeploymentRuntimeConfig]({{<ref "../concepts/providers#runtime-configuration">}}) 
-or a Composition using [patch and transforms]({{<ref "../concepts/patch-and-transform">}}) 
-to a 
-[Composition pipeline function]({{< ref "../concepts/compositions#use-composition-functions" >}}).
+or a legacy Composition using `mode: Resources` to a 
+[Composition pipeline function]({{< ref "../concepts/compositions" >}}).
```

Comment:
This should probably make clear that legacy Compositions that patch information into the XR `metadata` will not work properly after conversion and will need to be updated to use `status` instead.